### DISCUSSION TITLE
vimtex: passing default pdf viewer setting through LazyVim

### DISCUSSION THREAD

**Discussion by sh4k095** [LazyVim/LazyVim#643] [Q&A] [answered]
Hi everyone,

I'd like to add [lervag's vimtex](https://github.com/lervag/vimtex) to NeoVim through LazyVim. It's probably quite trivial, but I seemingly cannot pass the option `let g:vimtex_view_method="zathura"` to vimtex in order to use zathura as the default pdf viewer. This is: `lua/plugins/vimtex.lua`:

```
return {
  {
    "lervag/vimtex",
    config = function()
      vimtex_view_method = "zathura"
    end,
  },
}
```
The exact statement should be: `let g:vimtex_view_method="zathura"`, but I seemingly cannot pass more than one string to the function. Things like  `"let g:vimtex_view_method"="zathura"`,  `"let g:vimtex_view_method='zathura'"` etc. will also fail. Is this possible to overcome? Or is the syntax incompatible with LazyVim?

**Comment by folke** [comment, marked as answer]
use the `init` function for this instead of config. and do `vim.g.vimtex.... = "whatever"`

**Comment by sh4k095** [comment]
Amazing! That was it. Are there any "guidelines" on how to correctly pass plugin settings to LazyVim?

**Reply by folke** [reply]
check the lazy.nvim docs and probably also check the neovim lua guide
